internal/command/machine/clone.go

```diff
@@ -288,6 +290,15 @@ func runMachineClone(ctx context.Context) (err error) {
 
 	launchedMachine, err := flapsClient.Launch(ctx, input)
 	if err != nil {
+		if errors.As(err, &flaps.LaunchCapacityErr{}) {
+			return flyerr.GenericErr{
+				Err:      err.Error(),
+				Descript: flyerr.GetErrorDescription(err),
+				Suggest:  "Try cloning to a different nearby region, or try again in a few hours",
+				DocUrl:   flyerr.GetErrorDocUrl(err),
+			}
+
+		}
```

Comment:
Do you need to catch and wrap this? If `err` is a `flaps.LaunchCapacityErr` it should get picked up downstream and render the help text. You'd need to add a suggestion function though.

Reply:
nope, i'm just thinking about how i could add a suggestion per command